src/components/taskDetails/TaskContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, ReactNode } from 'react';
+import classNames from './task-details.module.scss';
+import Image from 'next/image';
+
```

Comment:
can we sort the imports here somewhat like in the format where
the library imports are first
then components
then CSS followed by assets

Reply:
done